python/tvm/relay/backend/contrib/ethosu/legalize.py

```diff
@@ -35,6 +35,40 @@
 from tvm.relay.op.contrib import ethosu as ethosu_patterns  # type: ignore
 
 
+def legalize_pass(
+    rewriter_cls: DFPatternCallback, name: Optional[str] = "", opt_level: Optional[int] = 1
```

Comment:
```suggestion
    rewriter_cls: DFPatternCallback, name: str = "", opt_level: int = 1
```
I think this is an incorrect usage of Optional, as far as I can tell the arguments will always be `str` and `int` but have default values if they are not given.